engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -2069,36 +2080,39 @@ static uintptr_t GetExtProcAddress(const char* name, const char* extension_name,
 
-    #ifdef GL_ES_VERSION_2_0
-        if(buffer_type_flags & dmGraphics::BUFFER_TYPE_COLOR0_BIT)
+        if (context->m_IsMRTSupported)
         {
-            uint32_t color_buffer_index = GetBufferTypeIndex(BUFFER_TYPE_COLOR0_BIT);
-            rt->m_ColorBufferTexture[0] = NewTexture(context, creation_params[color_buffer_index]);
-            SetTexture(rt->m_ColorBufferTexture[0], params[color_buffer_index]);
-            // attach the texture to FBO color attachment point
-            glFramebufferTexture2D(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0, GL_TEXTURE_2D, rt->m_ColorBufferTexture[0]->m_Texture, 0);
-            CHECK_GL_ERROR;
+            BufferType color_buffer_flags[] = {
+                BUFFER_TYPE_COLOR0_BIT,
+                BUFFER_TYPE_COLOR1_BIT,
+                BUFFER_TYPE_COLOR2_BIT,
+                BUFFER_TYPE_COLOR3_BIT,
+            };
+            for (int i = 0; i < MAX_BUFFER_COLOR_ATTACHMENTS; ++i)
+            {
+                if (buffer_type_flags & color_buffer_flags[i])
+                {
+                    uint32_t color_buffer_index = GetBufferTypeIndex(color_buffer_flags[i]);
+                    rt->m_ColorBufferTexture[i] = NewTexture(context, creation_params[color_buffer_index]);
+                    SetTexture(rt->m_ColorBufferTexture[i], params[color_buffer_index]);
+                    // attach the texture to FBO color attachment point
+                    glFramebufferTexture2D(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0 + i, GL_TEXTURE_2D, rt->m_ColorBufferTexture[i]->m_Texture, 0);
+                    CHECK_GL_ERROR;
+                }
+            }
         }
-    #else
-        BufferType color_buffer_flags[] = {
-            BUFFER_TYPE_COLOR0_BIT,
-            BUFFER_TYPE_COLOR1_BIT,
-            BUFFER_TYPE_COLOR2_BIT,
-            BUFFER_TYPE_COLOR3_BIT,
-        };
-        for (int i = 0; i < MAX_BUFFER_COLOR_ATTACHMENTS; ++i)
+        else
         {
-            if (buffer_type_flags & color_buffer_flags[i])
+            if(buffer_type_flags & dmGraphics::BUFFER_TYPE_COLOR0_BIT)
             {
-                uint32_t color_buffer_index = GetBufferTypeIndex(color_buffer_flags[i]);
-                rt->m_ColorBufferTexture[i] = NewTexture(context, creation_params[color_buffer_index]);
-                SetTexture(rt->m_ColorBufferTexture[i], params[color_buffer_index]);
+                uint32_t color_buffer_index = GetBufferTypeIndex(BUFFER_TYPE_COLOR0_BIT);
+                rt->m_ColorBufferTexture[0] = NewTexture(context, creation_params[color_buffer_index]);
+                SetTexture(rt->m_ColorBufferTexture[0], params[color_buffer_index]);
                 // attach the texture to FBO color attachment point
-                glFramebufferTexture2D(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0 + i, GL_TEXTURE_2D, rt->m_ColorBufferTexture[i]->m_Texture, 0);
+                glFramebufferTexture2D(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0, GL_TEXTURE_2D, rt->m_ColorBufferTexture[0]->m_Texture, 0);
```

Comment:
I don't see a difference here versus the first loop iteration above?
Perhaps use:
```
uint32_t num_color_buffers = context->m_IsMRTSupported ? 4 : 1;
```